### ISSUE TITLE
[A11y] Desktop/Mobile Collection Landing: Collection Title

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1880]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Collection Show

### Admin user?

No

### Issue Description

Collection Landing: https://staging.oregondigital.org/collections/gwilliams?locale=en

The Collection Title has the following Link Function error:
If an anchor element functions as a custom widget, it must have the appropriate ARIA widget role.

Accessible name: (no value)
URL: (no value)
Role: (no value)
Tab Index: (no value)
Aria attributes: (no value)

### Suggested Fix

Needs an accessible name.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

**Comment by CGillen** [comment]
@azle By collection title do you mean these links?
<img width="1180" alt="image" src="https://user-images.githubusercontent.com/11052958/130645092-60720794-d167-4168-910c-32c0a502a823.png">
In which case, this is a duplicate of #1747 

Or this span?
<img width="581" alt="image" src="https://user-images.githubusercontent.com/11052958/130645136-e91e75ce-95f9-4301-9785-ddf81d087098.png">

If it's neither of these, could you point out what link you're talking about?

**Comment by azle** [comment]
Yes, but looks like it has been fixed. I am not get that warning anymore. However, I am getting the following validation errors:

5 x Missing form label, see red icon tag
<img width="725" alt="Screen Shot 2021-12-15 at 9 25 21 AM" src="https://user-images.githubusercontent.com/8353860/146234928-d35ebf4b-d4b0-4ab0-8012-f906174588b5.png">

1 x Broken Aria Reference
<img width="956" alt="Screen Shot 2021-12-15 at 9 25 49 AM" src="https://user-images.githubusercontent.com/8353860/146235010-fb5ab42d-94f3-4d83-8f4c-07152edb2f55.png">

**Comment by azle** [comment]
Updated 1/6/2024

Accessibility Insights has 30 errors, needs fixed.

<img width="781" alt="Screenshot 2025-01-06 at 10 52 51 AM" src="https://github.com/user-attachments/assets/357741f2-c404-4942-92fe-fb1e4afb56a9" />
